### PR TITLE
feat(FN-1385): implement clamav as Azure Container App

### PR DESCRIPTION
**Pull request by OxonianCambion**
### Introduction

Azure Container Instances don't support static IPs or any other way of routing when in a private vnet. They also have some other terminal issues. This means our virus scanning won't deploy properly (can't deploy with the same name in Europe!!) and may break randomly (restarts don't preserve IPs)
We need to use another way to perform our virus scanning - Azure Container Apps cover it

This card covers
* Simple ACA implementation of Clam AV
### Resolution
* ACA using ClamAV image is created and wired up. ACI implementation removed.

### CODE REVIEW

File: infrastructure/resource_group_level/main.bicep [hunk 1]
```diff
@@ -318,7 +318,7 @@ var parametersMap = {
       addressPrefixes: ['172.16.40.0/22', '172.16.60.0/23']
       applicationGatewayCidr: '172.16.41.0/24'
       appServicePlanEgressPrefixCidr: '172.16.42.0/28'
-      aciCidr: '172.16.42.16/29'
+      acaCidr: '172.16.42.32/27'
```
Comment (OxonianCambion) [inline infrastructure/resource_group_level/main.bicep:321]: Azure Container environments need a /27 subnet size (32 IPs). (You need to start a /27 on a multiple of 32.)

File: infrastructure/resource_group_level/modules/vnet.bicep [hunk 2]
```diff
@@ -29,7 +29,7 @@ var appServicePlanEgressSubnetName = '${appServicePlanName}-app-service-plan-egr
 var gatewaySubnetName = '${environment}-gateway'
 var privateEndpointsSubnetName = '${environment}-private-endpoints'
 
-var aciSubnetName = '${environment}-aci'
+var acaSubnetName = '${environment}-aca'
```
Comment (OxonianCambion) [inline infrastructure/resource_group_level/modules/vnet.bicep:32]: I'm not certain of the best naming here - "aca" for azure container app environment seems ok. We could call it clamav, but I suppose the vnet doesn't care what's inside, just that it is an ACA environment.

File: infrastructure/resource_group_level/modules/clamav-aca.bicep [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    }
+    workloadProfiles: [
+      {
+        workloadProfileType: 'Consumption'
+        name: 'Consumption'
+      }
+    ]
+    appLogsConfiguration: {
+      destination: 'log-analytics'
+      logAnalyticsConfiguration: {
```
Comment (OxonianCambion) [inline infrastructure/resource_group_level/modules/clamav-aca.bicep:30]: I stuck log analytics (here) and Application Insights (below) in, as it was very easy to add.

File: .github/workflows/bicep_workflow_feature.yml [hunk 4]
```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - feat/stbr-infra
+      - feat/FN-1385/clamav-aca
```
Comment (ChristianMcc) [inline .github/workflows/bicep_workflow_feature.yml:7]: can delete now

**Review by ChristianMcc — APPROVED** [review]
LGTM, can just remove the workflow branch trigger. Thanks for working on this!

**Pull request merged**